Dockerfile

```diff
@@ -1,10 +1,9 @@
 #syntax=docker/dockerfile:1.4
-
 # Versions
 FROM php:8.2-fpm-alpine AS php_upstream
 FROM mlocati/php-extension-installer:2 AS php_extension_installer_upstream
 FROM composer/composer:2-bin AS composer_upstream
-FROM caddy:2-alpine AS caddy_upstream
+FROM nginx:stable-alpine3.17 AS nginx_upstream
```

Comment:
By curiosity, why do you replace Caddy with NGINX? Caddy supports Laravel very well (one of Caddy's maintainers is a Laravel developer), supports much more features than NGINX (HTTP/3, auto-HTTPS...), and allows a much simpler Docker setup.

Reply:
Because I don't know it verry well so I prefer manage nginx I've worked with. So I don't close this review until I studied and re add Caddy. Thank's for think about! :)

Reply:
Hi Kévin, I re add Caddy as web server. It's very useful and simple, thanks again!